core/src/labels/labels.cpp

```diff
@@ -314,14 +316,12 @@ void Labels::update(const View& _view, float _dt,
         if (label->options().repeatDistance == 0.f) {
             continue;
         }
-        TextLabel* textLabel = dynamic_cast<TextLabel*>(label);
-        if (!textLabel) { continue; }
-        repeatGroupSet.push_back(textLabel);
+        repeatGroupSet.push_back(label);
```

Comment:
yeah, the change was originally from having both AlfonsLabel and TextLabel. Though repeatDistance == 0.f ensures that only TextLabels are added.